doc/pgprobackup.xml

```diff
@@ -4290,6 +4430,120 @@ pg_probackup archive-get -B <replaceable>backup_dir</replaceable> --instance <re
         For details, see section <link linkend="pbk-archiving-options">Archiving Options</link>.
       </para>
     </refsect3>
+
+    <refsect3 id="pbk-catchup" xreflabel="catchup" condition="12+">
+      <title>catchup</title>
+      <programlisting>
+pg_probackup catchup -b <replaceable>catchup_mode</replaceable>
+--source-pgdata=<replaceable>path_to_pgdata_on_remote_server</replaceable>
+--destination-pgdata=<replaceable>path_to_local_dir</replaceable>
+[--help] [--stream] [-j <replaceable>num_threads</replaceable>]
+[-T <replaceable>OLDDIR</replaceable>=<replaceable>NEWDIR</replaceable>]
+[<replaceable>connection_options</replaceable>] [<replaceable>remote_options</replaceable>]
+</programlisting>
+      <para>
+        Creates a copy of a <productname>PostgreSQL</productname>
+        instance without using the backup catalog.
+
+      <variablelist>
+      <varlistentry>
+<term><option>-b <replaceable>catchup_mode</replaceable></option></term>
+<term><option>--backup-mode=<replaceable>catchup_mode</replaceable></option></term>
+      <listitem>
+      <para>
+        Specifies the catchup mode to use. Possible values are:
+
+      <itemizedlist spacing="compact">
+        <listitem>
+          <para>
+            <literal>FULL</literal> — creates a full copy of the <productname>PostgreSQL</productname> instance.
+          </para>
+        </listitem>
+        <listitem>
+          <para>
+            <literal>DELTA</literal> — reads all data files in the data directory and
+            creates an incremental copy for pages that have changed
+            since the destination database was shut down cleanly.
+          </para>
+        </listitem>
+        <listitem>
+          <para>
+            <literal>PTRACK</literal> — creates an incremental PTRACK backup tracking
```

Comment:
Такой текст пойдет?
PTRACK — tracking page changes on the fly,
          only copies pages that have changed since the point of divergence
          of the source and destination databases...

Reply:
Звучит норм!

> Такой текст пойдет?
> PTRACK — tracking page changes on the fly,
> only copies pages that have changed since the point of divergence
> of the source and destination databases...
Мне кажется, очень важно добавить, что и читаются тоже только изменившие страницы